spec/datadog/appsec/contrib/active_record/postgresql_adapter_spec.rb

```diff
@@ -65,6 +68,22 @@
     processor.finalize
   end
 
+  context 'when RASP is disabled' do
```

Comment:
I think overall style wise it's better to have all test cases (`it`'s) within the context. Then it will be easier to setup the stub

```ruby
context 'rasp enabled' do
  allow(Datadog::AppSec).to receive(:rasp_enabled?).and_return(true)
end

context 'rasp disabled' do
  allow(Datadog::AppSec).to receive(:rasp_enabled?).and_return(true)
end
```

Reply:
thanks for your comments, applied all of them [here](https://github.com/DataDog/dd-trace-rb/pull/4311/commits/87c14cdd8832b255c690cd666c0ef20700746cd7)